wgengine/magicsock/magicsock.go

```diff
@@ -1100,19 +1100,20 @@ func (a *AddrSet) UpdateDst(new *net.UDPAddr) error {
 	a.mu.Lock()
 	defer a.mu.Unlock()
 
-	if a.roamAddr != nil {
-		if equalUDPAddr(a.roamAddr, new) {
-			// Packet from the current roaming address, no logging.
-			// This is a hot path for established connections.
-			return nil
-		}
-	} else if a.curAddr >= 0 && equalUDPAddr(new, &a.addrs[a.curAddr]) {
+	if a.roamAddr != nil && equalUDPAddr(a.roamAddr, new) {
+		// Packet from the current roaming address, no logging.
+		// This is a hot path for established connections.
+		return nil
+	}
+	if a.curAddr >= 0 && equalUDPAddr(new, &a.addrs[a.curAddr]) {
```

Comment:
be consistent about whether `new` is the first or second argument to equalUDPAddr between line 1103 and here?